### PR TITLE
chore(core): Include StreamID in log message

### PR DESCRIPTION
**Pull request by stbrody**

### CODE REVIEW

**Review by oed — APPROVED** [review]

File: packages/common/src/utils/stream-utils.ts [hunk 1]
```diff
@@ -12,12 +12,21 @@ import {
 } from '../index'
 import { AnchorStatus, StreamState, LogEntry } from '../stream'
 import type { DagJWS } from 'dids'
-import { StreamType } from '@ceramicnetwork/streamid'
+import {StreamID, StreamType} from '@ceramicnetwork/streamid'
```
Comment (ukstv) [inline packages/common/src/utils/stream-utils.ts:15]: Prettier, please.

**Review by ukstv — CHANGES_REQUESTED** [review]
Prettier, please. WebStorm can do this for you.

**Comment by oed** [comment]
💅

**Review by smrz2001 — APPROVED** [review]
🚢

**Pull request merged**